tools/replay-viewer/style.css

```
@@ -0,0 +1,564 @@
+body {
+  min-height: 100vh;
+}
+
+.app-shell {
+  min-height: 100vh;
+}
+
+button,
+select,
+.file-input__button {
+  border: 1px solid rgba(148, 163, 184, 0.25);
```

Stylelint (17.11.0)

[error] Expected "0.25" to be "25%" (alpha-value-notation)

(alpha-value-notation)